openpype/hosts/houdini/api/creator_node_shelves.py

```diff
@@ -173,6 +173,7 @@ def install():
         os.remove(filepath)
 
     icon = get_openpype_icon_filepath()
+    tab_menu_label = os.environ.get("AVALON_LABEL") or "OpenPype"
```

Comment:
Is `AVALON_LABEL` always defined in Ayon? Since the rest seems refactored/default to Ayon instead but this still mentions OpenPype.

Reply:
It's defined in both OpenPype and Ayon. 
![image](https://github.com/ynput/OpenPype/assets/20871534/f35b21ab-9675-4ff8-a106-85f12cd5ae69)

Reply:
I see - since we're refactoring to Ayon the other keys by default - should we maybe also move the fallbacks to `OpenPype` here to AYON label?

Reply:
Makes more sense!